docs/commands/zq.md

```diff
@@ -421,7 +421,7 @@ echo <values> | zq <query> -
 which is used throughout the [language documentation](../language/README.md)
 and [operator reference](../language/operators/README.md).
 
-The language documentation and [tutorials directory](../tutorials)
+The language documentation and [tutorials directory](../tutorials/README.md)
```

Comment:
These link destinations that just point to a directory work ok when browsing the docs within GitHub itself, but on the deployed site they produce a "Page Not Found" error when clicked.